versioned_docs/version-0.x.0/kubeslice-cli/installing-kubeslice.mdx

```diff
@@ -0,0 +1,234 @@
+# Installing KubeSlice
+
+There are the five main steps for installing the KubeSlice:
+
+- Installing the KubeSlice Controller
+- Creating a project
+- Registering the worker clusters
+- Creating a slice
+- Onboarding namespaces onto the slices
+
+This topic describes the steps to install KubeSlice in your existing multi-cluster configuration.
+
+## Installing KubeSlice in a Multi-Cluster Configuration
+
+Kubeslice-cli is a tool that makes it simple to install Kubeslice into your existing clusters. The primary purpose of the tool is to install 
+kubeslice into your already existing clusters using the description of those clusters that you provide in a **custom topology** YAML file.
+Depending on the requirements, the Kubeslice Controller and/or worker components can be installed either incrementally (one cluster at a time) 
```

Comment:
It is ... KubeSlice Controller and/or worker components....

Reply:
updated